.github/workflows/build.yml

```diff
@@ -48,3 +48,12 @@ jobs:
         with:
           name: eduID-${{ github.sha }}-debug.apk
           path: app/build/outputs/apk/debug/app-debug.apk
+  check-tranlations:
```

Comment:
Shouldn't we run this before the build? So it doesn't build a half-translated app? Or is this just an extra check?
You could also run it in a separate workflow when doing a PR for example

Reply:
This runs on all PR's. An untranslated test-apk is not a big deal, just a 2do before release